manifests/init.pp

```diff
@@ -102,9 +105,9 @@
   $use_sympa           = false,         # postfix_use_sympa
   $postfix_ensure      = 'present',
   $mailx_ensure        = 'present',
-) inherits postfix::params {
-
+) { include ::postfix::params
```

Comment:
If params (where the template is set) is inherited we can't pass the chroot_daemons variable to it, as params is evaluated first due to the inheritance.